scripts/gcp/enable-gcp-apis.sh

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Default values
+DRY_RUN=false
+BATCH_SIZE=10
+PROJECT_ID=""
+
+# Usage information
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Enable all available GCP APIs for a project.
+
+OPTIONS:
+    -p, --project       GCP Project ID (optional, uses gcloud configured project if not provided)
+    -d, --dry-run       Show what would be enabled without actually enabling
+    -b, --batch-size    Number of APIs to enable at once (default: 10)
+    -h, --help          Show this help message
+
+EXAMPLES:
+    # Use configured project
+    $0
+    $0 --dry-run
+
+    # Or specify project explicitly
+    $0 --project my-project-id
+    $0 --project my-project-id --dry-run
+    $0 --project my-project-id --batch-size 20
+
+NOTES:
+    - Requires gcloud CLI to be installed and authenticated
+    - Enabling all APIs may have billing implications
+    - Some APIs may require additional setup after enabling
+
+EOF
+    exit 1
+}
+
+# Parse command line arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        -p|--project)
+            PROJECT_ID="$2"
+            shift 2
+            ;;
+        -d|--dry-run)
+            DRY_RUN=true
+            shift
+            ;;
+        -b|--batch-size)
+            BATCH_SIZE="$2"
+            shift 2
+            ;;
+        -h|--help)
+            usage
+            ;;
+        *)
+            echo -e "${RED}Unknown option: $1${NC}"
+            usage
+            ;;
+    esac
+done
+
+# Check if gcloud is installed
+if ! command -v gcloud &> /dev/null; then
+    echo -e "${RED}Error: gcloud CLI is not installed${NC}"
+    echo "Install from: https://cloud.google.com/sdk/docs/install"
+    exit 1
+fi
```

Comment:
I'd normally say to just skip this check and let bash and the host report the error, but things get weird in GH actions, so looks good to me.